source/logic/bounties.js

```diff
@@ -18,27 +19,26 @@ function setDB(database) {
 }
 
 /**
- * @param {Guild} guild
+ * @param {{slotNumber: number, posterId: string} | string} bountyInfo
+ * @param {Guild | undefined} guild
+ * @returns {Bounty | null} the bounty, if it exists
```

Comment:
This type can be abbreviated to `Bounty?`. I'm still on the fence about whether I like that better though.